components/atom/slider/src/index.js

```diff
@@ -151,7 +157,9 @@ AtomSlider.propTypes = {
   /* callback to format the value shown as label */
   valueLabelFormatter: PropTypes.func,
   /* flag to hide tooltip if wanted */
-  hideTooltip: PropTypes.bool
+  hideTooltip: PropTypes.bool,
+  /* If true it will invert the colors for selected track and rail */
+  invertColors: PropTypes.bool
```

Comment:
I know that what it technically does is invert colors but I think the prop of the component should describe not its technically use but its meaning, and it is to invert the evaluation from min to max or the other way around.

Reply:
i disagree since the slider its not technically a range, we are not inverting any evaluation, we are just using a visual change to give a different feeling/UX. I think its clearer for a prop name to describe what it does to the component without any assumption about the goal that you want to accomplish. 
Using another name like invertRange or something like this can cause confusion, since we are not doing any of that.